opendatakit/aggregate
# Issue 345 remove appengine support

Closes #345

#### What has been done to verify that this works as intended?
- Run Aggregate to verify it works
- Build the installer and execute it to verify it works (linux - postgresql)

#### Why is this the best possible solution? Were any other approaches considered?
N/A

#### Are there any risks to merging this code? If so, what are they?
Aggregate won't work on AppEngine any longer

#### Do we need any specific form for testing your changes? If so, please attach one
Nope.

#### Does this change require updates to documentation? If so, please file an issue at https://github.com/opendatakit/docs/issues/new and include the link below.
Yes. I'll create issues before the release
